### PR TITLE
k3d: add page

### PR DESCRIPTION
**Pull request by thamaraiselvam**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/k3d.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Delete a cluster:
+
+`k3d cluster delete {{cluster_name}}`
+
+- Create a new containerized k3s node:
+
+`k3d node create {{node_name}}`
+
+- Import a image from docker into k3d cluster:
```
Comment (kbdharun) [inline pages/common/k3d.md:18]:
```suggestion
- Import an image from docker into k3d [c]luster:
```
We prefer long arguments but when we use short single-character arguments we use mnemonics ([]) to show the long arguments.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/k3d.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# k3d
+
+> Wrapper CLI to easily create k3s clusters inside docker.
```
Comment (navarroaxel) [inline pages/common/k3d.md:3]:
```suggestion
> Wrapper CLI to easily create k3s clusters inside Docker.
```

File: pages/common/k3d.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Delete a cluster:
+
+`k3d cluster delete {{cluster_name}}`
+
+- Create a new containerized k3s node:
+
+`k3d node create {{node_name}}`
+
+- Import an image from docker into k3d [c]luster:
```
Comment (navarroaxel) [inline pages/common/k3d.md:18]:
```suggestion
- Import an image from Docker into a k3d cluster:
```

File: pages/common/k3d.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# k3d
+
+> Wrapper CLI to easily create k3s clusters inside docker.
+> More information: <https://k3d.io/>.
```
Comment (navarroaxel) [inline pages/common/k3d.md:4]:
```suggestion
> More information: <https://k3d.io>.
```

File: pages/common/k3d.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`k3d cluster delete {{cluster_name}}`
+
+- Create a new containerized k3s node:
+
+`k3d node create {{node_name}}`
+
+- Import an image from docker into k3d [c]luster:
+
+`k3d image import {{image_name}} -c {{cluster_name}}`
```
Comment (navarroaxel) [inline pages/common/k3d.md:20]: We prefer long arguments if available
```suggestion
`k3d image import {{image_name}} --cluster {{cluster_name}}`
```

**Review by kbdharun — APPROVED** [review]

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**